docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -404,12 +386,7 @@ For follow-up appointments only, a simulant has a probability of missing their a
 visits, it is assumed the patient seeks medical care. For screening visits, the chance to not attend 
 is covered by the probability of a visit. 
 
-The probability of missing a follow-up appointment is defined at the simulant level, and stays for 
-the duration of the sim. The probability is assigned to each sim based on a uniform distribution 
-between 5 and 35%. 
-
-When a simulant has a follow-up scheduled, they will have their assigned percent chance of missing that appointment. 
-[Parikh_2010]_ [Kheirkhah_2016]_ [Mohammadi_2018]_ [Denney_2019]_ [Kempny_2016]_ [McLeod_2005]_
+The probability of missing a follow-up appointment is 8.68% for all simulants. [Hwang_2015]_ 
```

Comment:
As discussed, ideally you'd only make changes pertinent to the PR title. That said, it happens, in which case it's probably safest to ping the engineers explicitly (especially in a situation like this where it's actually changing implementation)

Reply:
Makes sense - I will be more cognizant of this moving forward